libs/build.gradle

```diff
@@ -7,10 +7,42 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-configure(childProjects.values() - project('elasticsearch-log4j')) {
+configure(childProjects.values()) {
+
+  apply plugin: 'base'
+
   /*
-   * All subprojects are java projects using Elasticsearch's standard build
-   * tools.
+   * Although these libs are local to Elasticsearch, they can conflict with other similarly
```

Comment:
Is this still a compelling motivation, or at this point are we just maintaining this convention because changing it would be too much pain?

Reply:
I think it's still compelling. It's not _that_ difficult to maintain. And it allows us to have such simple names as "logging" that might conflict otherwise.

Reply:
It's also common, eg all lucene jars begin with `lucene-`.